uber/templates/signups/shifts.html

```diff
@@ -158,18 +157,13 @@
                 element.find('.fc-title').append('</br>' + buttonTag);
             }
         });
-        {% if 'schedule' in c.QUERY_STRING %}
-            $("#shift_cal").fullCalendar( 'changeView','listWeek');
-        {% else %}
-            // if opened with tgt_date, go to date in calendar
+        {% if view %}
```

Comment:
Instead of using jQuery to try to grab URL parameters, we just pass those parameters to the template and use if checks in Jinja2 to render or not render a change in view.